phpreport-report

```diff
@@ -321,10 +321,11 @@ class DetailedReport(Report):
         stories = []
         for task in tasks_for_day:
             story = ""
-            if (task.story):
+            if task.story:
                 story += "[" + task.story + "] "
             story += task.text
-            stories.append(story)
+            if len(stories) == 0 or story != stories[-1]:
```

Comment:
I prefer to do this in another way which I will do in a followup patch. Hopefully that's alright with you.